app/components/contributors-widget/template.hbs

```diff
@@ -1,17 +1,23 @@
 <table style="width: 100%;"class="list-widget widget-body">
```

Comment:
The plan is to refactor those widgets, but I think the refactor should happen in a separate, follow-up PR.